### ISSUE TITLE
/telescope-api blocked by VerifyCsrfToken middleware

### ISSUE THREAD

**Issue by itainathaniel** [laravel/telescope#82]
Laravel 5.7.10
Chrome, Safari, FireFox on secure Valet directory

After installing Telescope and browsing to `/telescope/requests`, the page shows **Scanning...** non stop.
On the console I get
```
Uncaught (in promise) TypeError: Cannot read property 'length' of undefined
    at app.js:79580
```
and that line is
```
_this3.lastEntryIndex = response.data.entries.length ? __WEBPACK_IMPORTED_MODULE_1_lodash___default.a.last(response.data.entries).sequence : _this3.lastEntryIndex;
```

When I did console.log() for `response.data`, it gave me the HTML content of `view('telescope::layout')`, and further tests revealed that all routes are ignored and my POST requests are turning to GET and fetching the fallback route:
```
Route::get('/{view?}', 'HomeController@index')->where('view', '(.*)');
```
![screen shot 2018-10-24 at 1 36 44](https://user-images.githubusercontent.com/971236/47394923-4f602000-d72d-11e8-83ac-f497bab718d9.png)

I've installed a fresh Laravel project and it all worked fine, so something is bothering Telescope on my working project, but I can't understand what it is and how POST request is turning to a GET request.

Ant thoughts?

**Comment by enimiste** [comment]
I checked my network console and i see something not normal.
The XHR request dont contain the host.
![2018-10-24_00h47_47](https://user-images.githubusercontent.com/2515677/47397307-9696e280-d726-11e8-94bd-2a60b7762b75.png)


The path in my config is :
`'path' => '/___/telescope',`

**Comment by itainathaniel** [comment]
When disabling `\App\Http\Middleware\VerifyCsrfToken::class,` in `App\Http\Kernel.php`, it works well, so it's a CSRF issue which I don't understand.

**Comment by itainathaniel** [comment]
@themsaid any ideas on tests and/or checks I could to to investigate?

**Comment by ForzaSFerrari** [comment]
I get it for example with Laravel Passport #64

**Comment by matalina** [comment]
The url for the api quire is wrong:  ```http://telescope-api/queries?tag=&before=&take=50&family_hash=...```

you can see it in @enimiste's screenshot there is no localhost, no domain what so ever.

I'm also having the same issue on a clean install of laravel and telescope.  I know the data is being recorded as there is data in the database, but nothing is being displayed because the Javascript is broken.

What url is the javascript using.  I have set APP_URL in my .env file.  Is there another place where the URL is taken?

**Comment by enimiste** [comment]
I did the fix at PR [#94] (https://github.com/laravel/telescope/pull/94)  but not accepted.

**Comment by themsaid** [comment]
duplicate of https://github.com/laravel/telescope/issues/64

**Comment by maggz69** [comment]
I ran into the same problem and realised that I had configured my laravel installation wrongly. I had set the session_domain as 127.0.0.1:8000 which meant that the auth cookie was being set on the full domain + port number. This brought about the error 419 when telescope tried to make requests since there wasnt a session cookie for 127.0.0.1. Another side effect was that requests would fail to due the VerifyCSRFGuard failing.

Changing the domain name to a valid name fixed the issue. Not sure is this is what @itainathaniel experienced as well.
